src/util/freeMemory.ts

```diff
@@ -0,0 +1,28 @@
+import PodManager from '../services/PodManager'
```

Comment:
I feel like freeMemory.ts is not the best name for this file since it also does server info. Something along the lines of serverInfo.ts which includes the freeMem function would probably be better.